## 01_automake1.10.1.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
## Description: Submit bugs to Debian instead of KDE.
Index: knmap-2.1/src/main.cpp
===================================================================
--- knmap-2.1.orig/src/main.cpp	2008-06-07 20:26:13.000000000 -0400
+++ knmap-2.1/src/main.cpp	2008-06-07 20:27:28.000000000 -0400
@@ -59,6 +59,7 @@
 					 "Initial rewrite released as version 1.0\nVersions 2.0, 2.1",
 					 "kev.gilbert@cdu.edu.au",
 					 "http://informatics.cdu.edu.au/staff/kgilbert" );
+    about.setBugAddress( "http://bugs.debian.org/" );
 	KCmdLineArgs::init( argc, argv, &about );
 	KCmdLineArgs::addCmdLineOptions( options );
 	KApplication app;
